config.yaml

```diff
@@ -5,7 +5,7 @@ description: AWS EC2 Container Service
 version: 0.0.1
 docker:
   image: cogcmd/aws-ecs
-  tag: "0.0.1"
+  tag: 0.0.1
```

Comment:
Did you verify that this installs correctly? I've gotten schema validation errors about this. Probably something we need to fix in spanner.